cstore_fdw--1.5.sql

```diff
@@ -58,3 +60,37 @@ CREATE EVENT TRIGGER cstore_drop_event
     ON SQL_DROP
     EXECUTE PROCEDURE cstore_drop_trigger();
 
+CREATE TABLE public.pg_cstore_tables (logicalrelid regclass PRIMARY KEY);
```

Comment:
Ok. makes sense. Assuming you agree with what I‌ suggested, I am not reviewing table footer related changes. Also I‌ am not reviewing style or comments and am keeping that as last step after we make the high level changes and test them.